ember-toucan-core/src/-private/components/error.hbs

```diff
@@ -18,5 +22,15 @@
     />
   </svg>
 
-  {{yield}}
+  {{#if this.hasMoreThanOneError}}
+    {{! Unfortunately we need this inline style to help with the ul offset  }}
+    {{! template-lint-disable no-inline-styles}}
+    <ul class="m-0 list-none space-y-1 p-0" style="margin-top: -2px">
```

Comment:
Without the negative-margin you get this 😭 

<img width="414" alt="Screenshot 2023-03-20 at 10 36 22 AM" src="https://user-images.githubusercontent.com/8069555/226373062-e1ef4255-8604-405d-a2ed-96eee7203d09.png">

Reply:
Fixed with https://github.com/CrowdStrike/ember-toucan-core/pull/83/commits/11b591e8913907e603276c59ddeb5499eca9913c